code/modules/antagonists/cult/cult_bastard_sword.dm

```diff
@@ -81,16 +82,10 @@
 /obj/item/cult_bastard/IsReflect(def_zone)
 	if(!spinning)
 		return FALSE
-	playsound(src, pick('sound/weapons/effects/ric1.ogg', 'sound/weapons/effects/ric2.ogg', 'sound/weapons/effects/ric3.ogg', 'sound/weapons/effects/ric4.ogg', 'sound/weapons/effects/ric5.ogg'), 100, 1)
 	return TRUE
 
 /obj/item/cult_bastard/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
 	if(!prob(final_block_chance))
 		return FALSE
-	if(attack_type == PROJECTILE_ATTACK)
-		owner.visible_message(span_danger("[owner] deflects [attack_text] with [src]!"))
-		playsound(src, pick('sound/weapons/effects/ric1.ogg', 'sound/weapons/effects/ric2.ogg', 'sound/weapons/effects/ric3.ogg', 'sound/weapons/effects/ric4.ogg', 'sound/weapons/effects/ric5.ogg'), 100, 1)
```

Comment:
We seem to be removing a lot of sounds here, is there a reason we want it gone?

Reply:
Now reflecting plays a blocking sound of the item, so this code is redundant.
As if it stayed, reflecting would've played 2 sounds at once.

Reply:
Also **only** cult weapons had this kind of fuckery